cp1404practicals/prac_6/languages.py

```diff
@@ -0,0 +1,22 @@
+# from folder_directory.filename import Class name
+from prac_6.programming_language import ProgrammingLanguage
+
+# This creates "instances" of the programming language class
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+
+# This will print True if python has dynamic typing (it does)
+print(ProgrammingLanguage.is_dynamic(python))
+
+# Prints all the information on this language
+print(python)
+
+# Print all languages that have dynamic typing
```

Comment:
Looks good. Nice use of comments to separate out the parts of the task and explain what each bit does